src/gc/config.d

```diff
@@ -63,6 +66,7 @@ struct Config
     profile:0|1|2  - enable profiling with summary when terminating program (%d)
     precise:0|1    - enable precise scanning (not implemented yet)
     concurrent:0|1 - enable concurrent collection (not implemented yet)
+    malloc:0|1     - enable a manual management implementation
```

Comment:
This idea doesn't scale to link new unknown GCs, let's use a string option `gc:conservative|malloc|concurrent` instead.